hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/WriteStatus.java

```diff
@@ -76,10 +78,15 @@ public class WriteStatus implements Serializable {
   private final boolean trackSuccessRecords;
   private final transient Random random;
 
-  public WriteStatus(Boolean trackSuccessRecords, Double failureFraction) {
+  public WriteStatus(Boolean trackSuccessRecords, Double failureFraction, Boolean isMetadataTable) {
```

Comment:
If `Boolean` is required be sure to handle the `null` case so you don't end up with an NPE at runtime

Reply:
We do have overloaded constructor right which passes false. 
Curious to understand how we can end up w/ `null` for the 3rd arg here.